src/app/(auth)/reset-password/page.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
```

Comment:
(1) We'll need to make some changes to this to ensure that the page only renders the reset password stuff if we're accessing this route via a magic link that is sent to the user via email. The steps for this are numbered for clarity!